src/normalize/mod.rs

```diff
@@ -11,12 +11,19 @@
 //!
 //! ```no_run
 //! use blazesym::normalize::Normalizer;
+//! use blazesym::Addr;
 //! use blazesym::Pid;
 //!
 //! let normalizer = Normalizer::new();
-//! let addrs = [0xdeadbeef];
-//! let pid = Pid::from(1234);
+//! let fopen_addr = libc::fopen as Addr;
+//! let addrs = [fopen_addr];
+//! let pid = Pid::Slf;
 //! let norm_addrs = normalizer.normalize_user_addrs(&addrs, pid).unwrap();
+//! assert_eq!(norm_addrs.addrs.len(), 1);
+//!
+//! let (addr, meta_idx) = norm_addrs.addrs[0];
+//! // fopen (0x7f5f8e23a790) is at address 0x77790 in Binary(Binary { path: "/usr/lib64/libc.so.6", build_id: Some([...]), ... })
```

Comment:
s/address/offset/ ? `0x7f5f8e23a790` is the address, no?

Reply:
Not sure about "proper" terminology, but in my mind both are addresses: the former is the virtual address while the other is the normalized / file address (in the binary). Not sure if offset is better, because at least in my head there is an association to file offset, which is about the byte position from the start of the file -- and that is not it. Let me know if you feel strongly and I can change it.

Reply:
Ok, fair enough about virt address. I'd maybe paraphrase this to "fopen (0x....) corresponds to 0x777790 address within Binary ...".

Reply:
Sounds good, will adjust.